phpstan.neon

```diff
@@ -0,0 +1,24 @@
+includes:
+    - vendor/jangregor/phpstan-prophecy/src/extension.neon
+    - vendor/phpstan/phpstan-symfony/extension.neon
+    - vendor/phpstan/phpstan-doctrine/extension.neon
+
+parameters:
+    level: 0
+    symfony:
+        container_xml_path: %rootDir%/../../../var/cache/admin/dev/adminDevDebugProjectContainer.xml
+    excludes_analyse:
+        - %currentWorkingDirectory%/src/Sulu/Bundle/*/DependencyInjection/Configuration.php
+        - %currentWorkingDirectory%/src/Sulu/Bundle/*/Tests/*
+        - %currentWorkingDirectory%/src/Sulu/Component/*/Tests/*
+        - %currentWorkingDirectory%/src/Sulu/Bundle/ContentBundle/Resources/phpcr-migrations/*
+        - %currentWorkingDirectory%/src/Sulu/Bundle/TestBundle/*
+        - %currentWorkingDirectory%/src/Sulu/Bundle/ResourceBundle/*
+        - %currentWorkingDirectory%/src/Sulu/Component/DocumentManager/Event/QueryCreateBuilderEvent.php
+        - %currentWorkingDirectory%/src/Sulu/Bundle/AudienceTargetingBundle/Controller/TargetGroupController.php
```

Comment:
Do you plan to keep these excludes? Which ones do you also keep in your projects?